components/list/masonry/src/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+
+class ListMasonry extends React.Component {
+  constructor (props) {
```

Comment:
i don't see that as a prop so i defined this as a const. 
thanks for the feedback!